linkedarraylists/MyArrayList.java

```diff
@@ -4,44 +4,54 @@
 import java.util.Objects;
 
 public class MyArrayList<T> implements MyList<T> {
-    private int index = 0;
-    private T[] array = (T[]) new Object[10];
-    private double loadFactor = 0.75;
-    private double inc = 1.5;
+    private int size = 0;
+    private final int DEFAULT_CAPACITY = 10;
+    private Object[] data;
+
+    MyArrayList() {
+        data = new Object[DEFAULT_CAPACITY];
+    }
 
     @Override
-    public T get(int i) {
-        return array[i];
+    public T get(int index) {
+        if (index < size) {
+            return (T) data[index];
```

Comment:
you do the check index with size twice
should create helper method which check index and throw exception